### ISSUE TITLE
Problem: "Home Assistant is starting, not everything will be available until it is finished."  on dj218j - Armada38x 

### ISSUE THREAD

**Issue by avol-io** [SynoCommunity/spksrc#4182]
### Setup
_Package Name:_  Home assistant
_Package Version:_  0.114.2-7

_NAS Model:_ dj218j
_NAS Architecture:_ Armada38x 

### Expected behavior
Application run correctly 

### Actual behavior
After #4171 the application start but when i complete the access to dashboard it give me this message: "Home Assistant is starting, not everything will be available until it is finished." and notthing happen.

#### Package log
 cat /var/packages/homeassistant/target/var/homeassistant.log

```
2020-09-16 10:33:04 INFO (MainThread) [homeassistant.bootstrap] Config directory: /volume1/@appstore/homeassistant/var/config
2020-09-16 10:33:04 INFO (SyncWorker_2) [homeassistant.loader] Loaded homeassistant from homeassistant.components.homeassistant
2020-09-16 10:33:04 INFO (MainThread) [homeassistant.setup] Setting up homeassistant
2020-09-16 10:33:04 INFO (MainThread) [homeassistant.setup] Setup of domain homeassistant took 0.0 seconds
2020-09-16 10:33:04 INFO (SyncWorker_0) [homeassistant.loader] Loaded persistent_notification from homeassistant.components.persistent_notification
2020-09-16 10:33:04 INFO (MainThread) [homeassistant.setup] Setting up persistent_notification
2020-09-16 10:33:04 INFO (MainThread) [homeassistant.setup] Setup of domain persistent_notification took 0.0 seconds
2020-09-16 10:33:05 INFO (SyncWorker_2) [homeassistant.loader] Loaded script from homeassistant.components.script
2020-09-16 10:33:05 INFO (SyncWorker_4) [homeassistant.loader] Loaded default_config from homeassistant.components.default_config
2020-09-16 10:33:05 INFO (SyncWorker_3) [homeassistant.loader] Loaded group from homeassistant.components.group
2020-09-16 10:33:05 INFO (SyncWorker_1) [homeassistant.loader] Loaded scene from homeassistant.components.scene
2020-09-16 10:33:05 INFO (SyncWorker_6) [homeassistant.loader] Loaded automation from homeassistant.components.automation
2020-09-16 10:33:05 INFO (SyncWorker_0) [homeassistant.loader] Loaded tts from homeassistant.components.tts
2020-09-16 10:33:05 INFO (SyncWorker_7) [homeassistant.loader] Loaded http from homeassistant.components.http
2020-09-16 10:33:05 INFO (SyncWorker_5) [homeassistant.loader] Loaded cloud from homeassistant.components.cloud
2020-09-16 10:33:05 INFO (SyncWorker_8) [homeassistant.loader] Loaded webhook from homeassistant.components.webhook
2020-09-16 10:33:05 INFO (SyncWorker_12) [homeassistant.loader] Loaded alexa from homeassistant.components.alexa
2020-09-16 10:33:05 INFO (SyncWorker_4) [homeassistant.loader] Loaded frontend from homeassistant.components.frontend
2020-09-16 10:33:05 INFO (SyncWorker_1) [homeassistant.loader] Loaded api from homeassistant.components.api
2020-09-16 10:33:05 INFO (SyncWorker_0) [homeassistant.loader] Loaded auth from homeassistant.components.auth
2020-09-16 10:33:05 INFO (SyncWorker_13) [homeassistant.loader] Loaded config from homeassistant.components.config
2020-09-16 10:33:05 INFO (SyncWorker_14) [homeassistant.loader] Loaded device_automation from homeassistant.components.device_automation
2020-09-16 10:33:05 INFO (SyncWorker_5) [homeassistant.loader] Loaded lovelace from homeassistant.components.lovelace
2020-09-16 10:33:05 INFO (SyncWorker_15) [homeassistant.loader] Loaded onboarding from homeassistant.components.onboarding
2020-09-16 10:33:05 INFO (SyncWorker_2) [homeassistant.loader] Loaded person from homeassistant.components.person
2020-09-16 10:33:05 INFO (SyncWorker_12) [homeassistant.loader] Loaded search from homeassistant.components.search
2020-09-16 10:33:05 INFO (SyncWorker_17) [homeassistant.loader] Loaded websocket_api from homeassistant.components.websocket_api
2020-09-16 10:33:06 INFO (SyncWorker_6) [homeassistant.loader] Loaded system_log from homeassistant.components.system_log
2020-09-16 10:33:06 INFO (SyncWorker_1) [homeassistant.loader] Loaded history from homeassistant.components.history
2020-09-16 10:33:06 INFO (SyncWorker_19) [homeassistant.loader] Loaded recorder from homeassistant.components.recorder
2020-09-16 10:33:06 INFO (SyncWorker_10) [homeassistant.loader] Loaded logbook from homeassistant.components.logbook
2020-09-16 10:33:06 INFO (SyncWorker_13) [homeassistant.loader] Loaded map from homeassistant.components.map
2020-09-16 10:33:06 INFO (SyncWorker_21) [homeassistant.loader] Loaded mobile_app from homeassistant.components.mobile_app
2020-09-16 10:33:06 INFO (SyncWorker_11) [homeassistant.loader] Loaded ssdp from homeassistant.components.ssdp
2020-09-16 10:33:06 INFO (SyncWorker_5) [homeassistant.loader] Loaded sun from homeassistant.components.sun
2020-09-16 10:33:06 INFO (SyncWorker_23) [homeassistant.loader] Loaded system_health from homeassistant.components.system_health
2020-09-16 10:33:06 INFO (SyncWorker_16) [homeassistant.loader] Loaded updater from homeassistant.components.updater
2020-09-16 10:33:06 INFO (SyncWorker_2) [homeassistant.loader] Loaded zeroconf from homeassistant.components.zeroconf
2020-09-16 10:33:06 INFO (SyncWorker_25) [homeassistant.loader] Loaded zone from homeassistant.components.zone
2020-09-16 10:33:06 INFO (SyncWorker_4) [homeassistant.loader] Loaded input_boolean from homeassistant.components.input_boolean
2020-09-16 10:33:06 INFO (SyncWorker_17) [homeassistant.loader] Loaded input_datetime from homeassistant.components.input_datetime
2020-09-16 10:33:06 INFO (SyncWorker_18) [homeassistant.loader] Loaded input_text from homeassistant.components.input_text
2020-09-16 10:33:06 INFO (SyncWorker_9) [homeassistant.loader] Loaded input_number from homeassistant.components.input_number
2020-09-16 10:33:06 INFO (SyncWorker_1) [homeassistant.loader] Loaded input_select from homeassistant.components.input_select
2020-09-16 10:33:06 INFO (MainThread) [homeassistant.bootstrap] Domains to be set up: {'system_log', 'auth', 'config', 'group', 'cloud', 'alexa', 'sun', 'automation', 'map', 'system_health', 'ssdp', 'input_number', 'default_config', 'mobile_app', 'updater', 'lovelace', 'recorder', 'http', 'tts', 'webhook', 'device_automation', 'input_boolean', 'zeroconf', 'script', 'input_select', 'api', 'websocket_api', 'person', 'history', 'input_text', 'logbook', 'onboarding', 'frontend', 'zone', 'input_datetime', 'search', 'scene'}
2020-09-16 10:33:06 INFO (MainThread) [homeassistant.bootstrap] Setting up logging: {'system_log', 'recorder'}
2020-09-16 10:33:07 INFO (MainThread) [homeassistant.setup] Setting up recorder
2020-09-16 10:33:07 INFO (MainThread) [homeassistant.setup] Setting up http
2020-09-16 10:33:07 INFO (MainThread) [homeassistant.setup] Setup of domain http took 0.0 seconds
2020-09-16 10:33:08 INFO (MainThread) [homeassistant.setup] Setting up system_log
2020-09-16 10:33:08 INFO (MainThread) [homeassistant.setup] Setup of domain system_log took 0.0 seconds
2020-09-16 10:33:08 WARNING (Recorder) [homeassistant.components.recorder] Ended unfinished session (id=1 from 2020-09-15 09:49:03.464530)
2020-09-16 10:33:08 INFO (MainThread) [homeassistant.setup] Setup of domain recorder took 0.4 seconds
2020-09-16 10:33:08 INFO (MainThread) [homeassistant.bootstrap] Setting up stage 1: {'device_automation', 'system_log', 'auth', 'config', 'onboarding', 'cloud', 'lovelace', 'alexa', 'api', 'frontend', 'http', 'websocket_api', 'person', 'search', 'webhook'}
2020-09-16 10:33:08 INFO (MainThread) [homeassistant.setup] Setting up device_automation
2020-09-16 10:33:08 INFO (MainThread) [homeassistant.setup] Setup of domain device_automation took 0.0 seconds
2020-09-16 10:33:09 INFO (MainThread) [homeassistant.setup] Setting up lovelace
2020-09-16 10:33:09 INFO (MainThread) [homeassistant.setup] Setting up person
2020-09-16 10:33:09 INFO (MainThread) [homeassistant.setup] Setting up auth
2020-09-16 10:33:09 INFO (MainThread) [homeassistant.setup] Setup of domain auth took 0.0 seconds
2020-09-16 10:33:10 INFO (MainThread) [homeassistant.setup] Setting up config
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setting up alexa
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain alexa took 0.0 seconds
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setting up api
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setting up websocket_api
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain websocket_api took 0.0 seconds
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setting up webhook
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain webhook took 0.0 seconds
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain lovelace took 1.9 seconds
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain api took 0.1 seconds
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain config took 1.5 seconds
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setting up search
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain search took 0.0 seconds
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain person took 1.8 seconds
2020-09-16 10:33:11 INFO (SyncWorker_33) [homeassistant.loader] Loaded google_assistant from homeassistant.components.google_assistant
2020-09-16 10:33:11 INFO (SyncWorker_34) [homeassistant.loader] Loaded camera from homeassistant.components.camera
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setting up onboarding
2020-09-16 10:33:11 INFO (MainThread) [homeassistant.setup] Setup of domain onboarding took 0.0 seconds
2020-09-16 10:33:11 INFO (SyncWorker_36) [homeassistant.loader] Loaded media_player from homeassistant.components.media_player
2020-09-16 10:33:11 INFO (SyncWorker_3) [homeassistant.loader] Loaded device_tracker from homeassistant.components.device_tracker
2020-09-16 10:33:12 ERROR (MainThread) [homeassistant.setup] Setup failed for cloud: Unable to import component: No module named 'botocore'
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up frontend
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setup of domain frontend took 0.0 seconds
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.bootstrap] Setting up stage 2: {'group', 'sun', 'automation', 'map', 'system_health', 'ssdp', 'input_number', 'default_config', 'mobile_app', 'updater', 'tts', 'input_boolean', 'zeroconf', 'script', 'input_select', 'history', 'input_text', 'logbook', 'zone', 'input_datetime', 'scene'}
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up group
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setup of domain group took 0.0 seconds
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up sun
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setup of domain sun took 0.1 seconds
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up automation
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setup of domain automation took 0.0 seconds
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up input_number
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up updater
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setup of domain updater took 0.0 seconds
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up input_boolean
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up script
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setup of domain script took 0.0 seconds
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up input_select
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up input_text
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up zone
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up input_datetime
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.setup] Setting up scene
2020-09-16 10:33:12 INFO (MainThread) [homeassistant.components.scene] Setting up scene.homeassistant
2020-09-16 10:33:13 INFO (SyncWorker_42) [homeassistant.loader] Loaded binary_sensor from homeassistant.components.binary_sensor
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setting up map
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain map took 0.0 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setting up system_health
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain system_health took 0.0 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setting up history
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain history took 0.0 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setting up logbook
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain scene took 0.3 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain input_number took 1.0 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain input_boolean took 0.4 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain input_select took 0.4 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain input_text took 0.4 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setting up binary_sensor
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain binary_sensor took 0.0 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain input_datetime took 0.4 seconds
2020-09-16 10:33:13 INFO (SyncWorker_1) [homeassistant.loader] Loaded google_translate from homeassistant.components.google_translate
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setting up zeroconf
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setup of domain logbook took 0.7 seconds
2020-09-16 10:33:13 INFO (MainThread) [homeassistant.setup] Setting up tts
2020-09-16 10:33:14 INFO (MainThread) [homeassistant.setup] Setup of domain zone took 1.1 seconds
2020-09-16 10:33:14 INFO (MainThread) [homeassistant.components.binary_sensor] Setting up binary_sensor.updater
2020-09-16 10:33:14 INFO (MainThread) [homeassistant.setup] Setup of domain tts took 0.0 seconds
2020-09-16 10:33:14 INFO (MainThread) [homeassistant.setup] Setup of domain zeroconf took 0.2 seconds
2020-09-16 10:33:14 INFO (MainThread) [homeassistant.setup] Setting up ssdp
2020-09-16 10:33:14 INFO (MainThread) [homeassistant.setup] Setup of domain ssdp took 0.0 seconds
2020-09-16 10:33:15 INFO (MainThread) [homeassistant.components.updater] Submitted analytics to Home Assistant servers. Information submitted includes {}
2020-09-16 10:38:06 INFO (SyncWorker_61) [homeassistant.loader] Loaded cast from homeassistant.components.cast
2020-09-16 10:38:06 INFO (SyncWorker_27) [homeassistant.loader] Loaded vilfo from homeassistant.components.vilfo
2020-09-16 10:38:06 INFO (SyncWorker_39) [homeassistant.loader] Loaded vizio from homeassistant.components.vizio
2020-09-16 10:38:06 INFO (SyncWorker_40) [homeassistant.loader] Loaded plum_lightpad from homeassistant.components.plum_lightpad
2020-09-16 10:38:06 INFO (SyncWorker_60) [homeassistant.loader] Loaded panasonic_viera from homeassistant.components.panasonic_viera
2020-09-16 10:38:06 INFO (SyncWorker_6) [homeassistant.loader] Loaded wemo from homeassistant.components.wemo
2020-09-16 10:38:06 INFO (SyncWorker_61) [homeassistant.loader] Loaded cert_expiry from homeassistant.components.cert_expiry
2020-09-16 10:38:06 INFO (SyncWorker_54) [homeassistant.loader] Loaded august from homeassistant.components.august
2020-09-16 10:38:06 INFO (SyncWorker_18) [homeassistant.loader] Loaded wled from homeassistant.components.wled
2020-09-16 10:38:06 INFO (SyncWorker_43) [homeassistant.loader] Loaded logi_circle from homeassistant.components.logi_circle
2020-09-16 10:38:06 INFO (SyncWorker_9) [homeassistant.loader] Loaded dexcom from homeassistant.components.dexcom
2020-09-16 10:38:06 INFO (SyncWorker_10) [homeassistant.loader] Loaded luftdaten from homeassistant.components.luftdaten
2020-09-16 10:38:06 INFO (SyncWorker_63) [homeassistant.loader] Loaded squeezebox from homeassistant.components.squeezebox
2020-09-16 10:38:06 INFO (SyncWorker_19) [homeassistant.loader] Loaded neato from homeassistant.components.neato
2020-09-16 10:38:06 INFO (SyncWorker_31) [homeassistant.loader] Loaded dialogflow from homeassistant.components.dialogflow
2020-09-16 10:38:06 INFO (SyncWorker_41) [homeassistant.loader] Loaded flick_electric from homeassistant.components.flick_electric
2020-09-16 10:38:06 INFO (SyncWorker_29) [homeassistant.loader] Loaded blebox from homeassistant.components.blebox
2020-09-16 10:38:06 INFO (SyncWorker_17) [homeassistant.loader] Loaded freebox from homeassistant.components.freebox
2020-09-16 10:38:06 INFO (SyncWorker_42) [homeassistant.loader] Loaded samsungtv from homeassistant.components.samsungtv
2020-09-16 10:38:06 INFO (SyncWorker_48) [homeassistant.loader] Loaded myq from homeassistant.components.myq
2020-09-16 10:38:06 INFO (SyncWorker_62) [homeassistant.loader] Loaded zerproc from homeassistant.components.zerproc
2020-09-16 10:38:06 INFO (SyncWorker_13) [homeassistant.loader] Loaded wiffi from homeassistant.components.wiffi
2020-09-16 10:38:06 INFO (SyncWorker_7) [homeassistant.loader] Loaded meteo_france from homeassistant.components.meteo_france
2020-09-16 10:38:06 INFO (SyncWorker_22) [homeassistant.loader] Loaded bond from homeassistant.components.bond
2020-09-16 10:38:06 INFO (SyncWorker_28) [homeassistant.loader] Loaded nexia from homeassistant.components.nexia
2020-09-16 10:38:06 INFO (SyncWorker_30) [homeassistant.loader] Loaded hisense_aehw4a1 from homeassistant.components.hisense_aehw4a1
2020-09-16 10:38:06 INFO (SyncWorker_37) [homeassistant.loader] Loaded abode from homeassistant.components.abode
2020-09-16 10:38:06 INFO (SyncWorker_21) [homeassistant.loader] Loaded locative from homeassistant.components.locative
2020-09-16 10:38:06 INFO (SyncWorker_45) [homeassistant.loader] Loaded fritzbox from homeassistant.components.fritzbox
2020-09-16 10:38:06 INFO (SyncWorker_36) [homeassistant.loader] Loaded nuheat from homeassistant.components.nuheat
2020-09-16 10:38:06 INFO (SyncWorker_48) [homeassistant.loader] Loaded ps4 from homeassistant.components.ps4
2020-09-16 10:38:06 INFO (SyncWorker_22) [homeassistant.loader] Loaded wolflink from homeassistant.components.wolflink
2020-09-16 10:38:06 INFO (SyncWorker_30) [homeassistant.loader] Loaded geonetnz_quakes from homeassistant.components.geonetnz_quakes
2020-09-16 10:38:06 INFO (SyncWorker_20) [homeassistant.loader] Loaded tplink from homeassistant.components.tplink
2020-09-16 10:38:06 INFO (SyncWorker_22) [homeassistant.loader] Loaded azure_devops from homeassistant.components.azure_devops
2020-09-16 10:38:06 INFO (SyncWorker_37) [homeassistant.loader] Loaded netatmo from homeassistant.components.netatmo
2020-09-16 10:38:06 INFO (SyncWorker_54) [homeassistant.loader] Loaded isy994 from homeassistant.components.isy994
2020-09-16 10:38:06 INFO (SyncWorker_48) [homeassistant.loader] Loaded homematicip_cloud from homeassistant.components.homematicip_cloud
2020-09-16 10:38:06 INFO (SyncWorker_46) [homeassistant.loader] Loaded hangouts from homeassistant.components.hangouts
2020-09-16 10:38:06 INFO (SyncWorker_55) [homeassistant.loader] Loaded ios from homeassistant.components.ios
2020-09-16 10:38:06 INFO (SyncWorker_17) [homeassistant.loader] Loaded coronavirus from homeassistant.components.coronavirus
2020-09-16 10:38:06 INFO (SyncWorker_59) [homeassistant.loader] Loaded homekit from homeassistant.components.homekit
2020-09-16 10:38:06 INFO (SyncWorker_13) [homeassistant.loader] Loaded flume from homeassistant.components.flume
2020-09-16 10:38:06 INFO (SyncWorker_62) [homeassistant.loader] Loaded toon from homeassistant.components.toon
2020-09-16 10:38:06 INFO (SyncWorker_9) [homeassistant.loader] Loaded atag from homeassistant.components.atag
2020-09-16 10:38:06 INFO (SyncWorker_41) [homeassistant.loader] Loaded powerwall from homeassistant.components.powerwall
2020-09-16 10:38:06 INFO (SyncWorker_48) [homeassistant.loader] Loaded hvv_departures from homeassistant.components.hvv_departures
2020-09-16 10:38:06 INFO (SyncWorker_57) [homeassistant.loader] Loaded tile from homeassistant.components.tile
2020-09-16 10:38:06 INFO (SyncWorker_0) [homeassistant.loader] Loaded synology_dsm from homeassistant.components.synology_dsm
2020-09-16 10:38:06 INFO (SyncWorker_22) [homeassistant.loader] Loaded control4 from homeassistant.components.control4
2020-09-16 10:38:06 INFO (SyncWorker_47) [homeassistant.loader] Loaded metoffice from homeassistant.components.metoffice
2020-09-16 10:38:06 INFO (SyncWorker_45) [homeassistant.loader] Loaded garmin_connect from homeassistant.components.garmin_connect
2020-09-16 10:38:06 INFO (SyncWorker_53) [homeassistant.loader] Loaded tellduslive from homeassistant.components.tellduslive
2020-09-16 10:38:06 INFO (SyncWorker_21) [homeassistant.loader] Loaded melcloud from homeassistant.components.melcloud
2020-09-16 10:38:06 INFO (SyncWorker_36) [homeassistant.loader] Loaded tado from homeassistant.components.tado
2020-09-16 10:38:06 INFO (SyncWorker_43) [homeassistant.loader] Loaded glances from homeassistant.components.glances
2020-09-16 10:38:06 INFO (SyncWorker_8) [homeassistant.loader] Loaded nut from homeassistant.components.nut
2020-09-16 10:38:06 INFO (SyncWorker_2) [homeassistant.loader] Loaded poolsense from homeassistant.components.poolsense
2020-09-16 10:38:06 INFO (SyncWorker_58) [homeassistant.loader] Loaded ifttt from homeassistant.components.ifttt
2020-09-16 10:38:06 INFO (SyncWorker_27) [homeassistant.loader] Loaded plaato from homeassistant.components.plaato
2020-09-16 10:38:06 INFO (SyncWorker_11) [homeassistant.loader] Loaded songpal from homeassistant.components.songpal
2020-09-16 10:38:06 INFO (SyncWorker_24) [homeassistant.loader] Loaded doorbird from homeassistant.components.doorbird
2020-09-16 10:38:06 INFO (SyncWorker_1) [homeassistant.loader] Loaded upb from homeassistant.components.upb
2020-09-16 10:38:06 INFO (SyncWorker_18) [homeassistant.loader] Loaded tuya from homeassistant.components.tuya
2020-09-16 10:38:06 INFO (SyncWorker_33) [homeassistant.loader] Loaded gpslogger from homeassistant.components.gpslogger
2020-09-16 10:38:06 INFO (SyncWorker_11) [homeassistant.loader] Loaded gogogate2 from homeassistant.components.gogogate2
2020-09-16 10:38:06 INFO (SyncWorker_48) [homeassistant.loader] Loaded solarlog from homeassistant.components.solarlog
2020-09-16 10:38:06 INFO (SyncWorker_56) [homeassistant.loader] Loaded icloud from homeassistant.components.icloud
2020-09-16 10:38:06 INFO (SyncWorker_12) [homeassistant.loader] Loaded deconz from homeassistant.components.deconz
2020-09-16 10:38:06 INFO (SyncWorker_25) [homeassistant.loader] Loaded rachio from homeassistant.components.rachio
2020-09-16 10:38:06 INFO (SyncWorker_38) [homeassistant.loader] Loaded home_connect from homeassistant.components.home_connect
2020-09-16 10:38:06 INFO (SyncWorker_38) [homeassistant.loader] Loaded tradfri from homeassistant.components.tradfri
2020-09-16 10:38:06 INFO (SyncWorker_37) [homeassistant.loader] Loaded transmission from homeassistant.components.transmission
2020-09-16 10:38:06 INFO (SyncWorker_12) [homeassistant.loader] Loaded spider from homeassistant.components.spider
2020-09-16 10:38:06 INFO (SyncWorker_44) [homeassistant.loader] Loaded twilio from homeassistant.components.twilio
2020-09-16 10:38:06 INFO (SyncWorker_48) [homeassistant.loader] Loaded onvif from homeassistant.components.onvif
2020-09-16 10:38:06 INFO (SyncWorker_14) [homeassistant.loader] Loaded hunterdouglas_powerview from homeassistant.components.hunterdouglas_powerview
2020-09-16 10:38:06 INFO (SyncWorker_47) [homeassistant.loader] Loaded adguard from homeassistant.components.adguard
2020-09-16 10:38:06 INFO (SyncWorker_13) [homeassistant.loader] Loaded roomba from homeassistant.components.roomba
2020-09-16 10:38:06 INFO (SyncWorker_3) [homeassistant.loader] Loaded coolmaster from homeassistant.components.coolmaster
2020-09-16 10:38:06 INFO (SyncWorker_38) [homeassistant.loader] Loaded shopping_list from homeassistant.components.shopping_list
2020-09-16 10:38:06 INFO (SyncWorker_56) [homeassistant.loader] Loaded zha from homeassistant.components.zha
2020-09-16 10:38:06 INFO (SyncWorker_37) [homeassistant.loader] Loaded avri from homeassistant.components.avri
2020-09-16 10:38:06 INFO (SyncWorker_35) [homeassistant.loader] Loaded acmeda from homeassistant.components.acmeda
2020-09-16 10:38:06 INFO (SyncWorker_25) [homeassistant.loader] Loaded sonos from homeassistant.components.sonos
2020-09-16 10:38:06 INFO (SyncWorker_49) [homeassistant.loader] Loaded enocean from homeassistant.components.enocean
2020-09-16 10:38:06 INFO (SyncWorker_13) [homeassistant.loader] Loaded denonavr from homeassistant.components.denonavr
2020-09-16 10:38:06 INFO (SyncWorker_23) [homeassistant.loader] Loaded izone from homeassistant.components.izone
2020-09-16 10:38:06 INFO (SyncWorker_44) [homeassistant.loader] Loaded mqtt from homeassistant.components.mqtt
2020-09-16 10:38:06 INFO (SyncWorker_30) [homeassistant.loader] Loaded esphome from homeassistant.components.esphome
2020-09-16 10:38:06 INFO (SyncWorker_16) [homeassistant.loader] Loaded arcam_fmj from homeassistant.components.arcam_fmj
2020-09-16 10:38:06 INFO (SyncWorker_31) [homeassistant.loader] Loaded traccar from homeassistant.components.traccar
2020-09-16 10:38:06 INFO (SyncWorker_28) [homeassistant.loader] Loaded xiaomi_miio from homeassistant.components.xiaomi_miio
2020-09-16 10:38:06 INFO (SyncWorker_26) [homeassistant.loader] Loaded starline from homeassistant.components.starline
2020-09-16 10:38:06 INFO (SyncWorker_10) [homeassistant.loader] Loaded plex from homeassistant.components.plex
2020-09-16 10:38:06 INFO (SyncWorker_7) [homeassistant.loader] Loaded axis from homeassistant.components.axis
2020-09-16 10:38:06 INFO (SyncWorker_10) [homeassistant.loader] Loaded point from homeassistant.components.point
2020-09-16 10:38:06 INFO (SyncWorker_26) [homeassistant.loader] Loaded elgato from homeassistant.components.elgato
2020-09-16 10:38:06 INFO (SyncWorker_10) [homeassistant.loader] Loaded smhi from homeassistant.components.smhi
2020-09-16 10:38:06 INFO (SyncWorker_28) [homeassistant.loader] Loaded awair from homeassistant.components.awair
2020-09-16 10:38:06 INFO (SyncWorker_5) [homeassistant.loader] Loaded ipp from homeassistant.components.ipp
2020-09-16 10:38:06 INFO (SyncWorker_5) [homeassistant.loader] Loaded daikin from homeassistant.components.daikin
2020-09-16 10:38:06 INFO (SyncWorker_47) [homeassistant.loader] Loaded smarthab from homeassistant.components.smarthab
2020-09-16 10:38:06 INFO (SyncWorker_17) [homeassistant.loader] Loaded totalconnect from homeassistant.components.totalconnect
2020-09-16 10:38:06 INFO (SyncWorker_18) [homeassistant.loader] Loaded geonetnz_volcano from homeassistant.components.geonetnz_volcano
2020-09-16 10:38:06 INFO (SyncWorker_31) [homeassistant.loader] Loaded nest from homeassistant.components.nest
2020-09-16 10:38:06 INFO (SyncWorker_33) [homeassistant.loader] Loaded somfy from homeassistant.components.somfy
2020-09-16 10:38:06 INFO (SyncWorker_24) [homeassistant.loader] Loaded tibber from homeassistant.components.tibber
2020-09-16 10:38:06 INFO (SyncWorker_6) [homeassistant.loader] Loaded lifx from homeassistant.components.lifx
2020-09-16 10:38:06 INFO (SyncWorker_32) [homeassistant.loader] Loaded sonarr from homeassistant.components.sonarr
2020-09-16 10:38:07 INFO (SyncWorker_51) [homeassistant.loader] Loaded geofency from homeassistant.components.geofency
2020-09-16 10:38:07 INFO (SyncWorker_5) [homeassistant.loader] Loaded devolo_home_control from homeassistant.components.devolo_home_control
2020-09-16 10:38:07 INFO (SyncWorker_51) [homeassistant.loader] Loaded directv from homeassistant.components.directv
2020-09-16 10:38:07 INFO (SyncWorker_33) [homeassistant.loader] Loaded bsblan from homeassistant.components.bsblan
2020-09-16 10:38:07 INFO (SyncWorker_11) [homeassistant.loader] Loaded mill from homeassistant.components.mill
2020-09-16 10:38:07 INFO (SyncWorker_50) [homeassistant.loader] Loaded sentry from homeassistant.components.sentry
2020-09-16 10:38:07 INFO (SyncWorker_5) [homeassistant.loader] Loaded blink from homeassistant.components.blink
2020-09-16 10:38:07 INFO (SyncWorker_11) [homeassistant.loader] Loaded xiaomi_aqara from homeassistant.components.xiaomi_aqara
2020-09-16 10:38:07 INFO (SyncWorker_50) [homeassistant.loader] Loaded pi_hole from homeassistant.components.pi_hole
2020-09-16 10:38:07 INFO (SyncWorker_5) [homeassistant.loader] Loaded emulated_roku from homeassistant.components.emulated_roku
2020-09-16 10:38:07 INFO (SyncWorker_15) [homeassistant.loader] Loaded flunearyou from homeassistant.components.flunearyou
2020-09-16 10:38:07 INFO (SyncWorker_19) [homeassistant.loader] Loaded hlk_sw16 from homeassistant.components.hlk_sw16
2020-09-16 10:38:07 INFO (SyncWorker_50) [homeassistant.loader] Loaded roku from homeassistant.components.roku
2020-09-16 10:38:07 INFO (SyncWorker_24) [homeassistant.loader] Loaded almond from homeassistant.components.almond
2020-09-16 10:38:07 INFO (SyncWorker_47) [homeassistant.loader] Loaded griddy from homeassistant.components.griddy
2020-09-16 10:38:07 INFO (SyncWorker_18) [homeassistant.loader] Loaded minecraft_server from homeassistant.components.minecraft_server
2020-09-16 10:38:07 INFO (SyncWorker_33) [homeassistant.loader] Loaded elkm1 from homeassistant.components.elkm1
2020-09-16 10:38:07 INFO (SyncWorker_52) [homeassistant.loader] Loaded heos from homeassistant.components.heos
2020-09-16 10:38:07 INFO (SyncWorker_6) [homeassistant.loader] Loaded ovo_energy from homeassistant.components.ovo_energy
2020-09-16 10:38:07 INFO (SyncWorker_29) [homeassistant.loader] Loaded velbus from homeassistant.components.velbus
2020-09-16 10:38:07 INFO (SyncWorker_15) [homeassistant.loader] Loaded ipma from homeassistant.components.ipma
2020-09-16 10:38:07 INFO (SyncWorker_5) [homeassistant.loader] Loaded homekit_controller from homeassistant.components.homekit_controller
2020-09-16 10:38:07 INFO (SyncWorker_18) [homeassistant.loader] Loaded twentemilieu from homeassistant.components.twentemilieu
2020-09-16 10:38:07 INFO (SyncWorker_3) [homeassistant.loader] Loaded life360 from homeassistant.components.life360
2020-09-16 10:38:07 INFO (SyncWorker_54) [homeassistant.loader] Loaded mikrotik from homeassistant.components.mikrotik
2020-09-16 10:38:07 INFO (SyncWorker_7) [homeassistant.loader] Loaded smappee from homeassistant.components.smappee
2020-09-16 10:38:07 INFO (SyncWorker_10) [homeassistant.loader] Loaded ozw from homeassistant.components.ozw
2020-09-16 10:38:07 INFO (SyncWorker_59) [homeassistant.loader] Loaded zwave from homeassistant.components.zwave
2020-09-16 10:38:07 INFO (SyncWorker_20) [homeassistant.loader] Loaded pvpc_hourly_pricing from homeassistant.components.pvpc_hourly_pricing
2020-09-16 10:38:07 INFO (SyncWorker_32) [homeassistant.loader] Loaded speedtestdotnet from homeassistant.components.speedtestdotnet
2020-09-16 10:38:07 INFO (SyncWorker_35) [homeassistant.loader] Loaded hue from homeassistant.components.hue
2020-09-16 10:38:07 INFO (SyncWorker_16) [homeassistant.loader] Loaded dynalite from homeassistant.components.dynalite
2020-09-16 10:38:07 INFO (SyncWorker_26) [homeassistant.loader] Loaded withings from homeassistant.components.withings
2020-09-16 10:38:07 INFO (SyncWorker_62) [homeassistant.loader] Loaded juicenet from homeassistant.components.juicenet
2020-09-16 10:38:07 INFO (SyncWorker_4) [homeassistant.loader] Loaded huawei_lte from homeassistant.components.huawei_lte
2020-09-16 10:38:07 INFO (SyncWorker_24) [homeassistant.loader] Loaded unifi from homeassistant.components.unifi
2020-09-16 10:38:07 INFO (SyncWorker_45) [homeassistant.loader] Loaded soma from homeassistant.components.soma
2020-09-16 10:38:07 INFO (SyncWorker_51) [homeassistant.loader] Loaded plugwise from homeassistant.components.plugwise
2020-09-16 10:38:07 INFO (SyncWorker_37) [homeassistant.loader] Loaded gios from homeassistant.components.gios
2020-09-16 10:38:07 INFO (SyncWorker_11) [homeassistant.loader] Loaded iaqualink from homeassistant.components.iaqualink
2020-09-16 10:38:07 INFO (SyncWorker_56) [homeassistant.loader] Loaded notion from homeassistant.components.notion
2020-09-16 10:38:07 INFO (SyncWorker_21) [homeassistant.loader] Loaded ambient_station from homeassistant.components.ambient_station
2020-09-16 10:38:07 INFO (SyncWorker_12) [homeassistant.loader] Loaded rainmachine from homeassistant.components.rainmachine
2020-09-16 10:38:07 INFO (SyncWorker_36) [homeassistant.loader] Loaded harmony from homeassistant.components.harmony
2020-09-16 10:38:07 INFO (SyncWorker_31) [homeassistant.loader] Loaded vera from homeassistant.components.vera
2020-09-16 10:38:07 INFO (SyncWorker_17) [homeassistant.loader] Loaded mailgun from homeassistant.components.mailgun
2020-09-16 10:38:07 INFO (SyncWorker_13) [homeassistant.loader] Loaded vesync from homeassistant.components.vesync
2020-09-16 10:38:07 INFO (SyncWorker_44) [homeassistant.loader] Loaded iqvia from homeassistant.components.iqvia
2020-09-16 10:38:07 INFO (SyncWorker_47) [homeassistant.loader] Loaded simplisafe from homeassistant.components.simplisafe
2020-09-16 10:38:07 INFO (SyncWorker_48) [homeassistant.loader] Loaded dunehd from homeassistant.components.dunehd
2020-09-16 10:38:07 INFO (SyncWorker_28) [homeassistant.loader] Loaded braviatv from homeassistant.components.braviatv
2020-09-16 10:38:07 INFO (SyncWorker_30) [homeassistant.loader] Loaded owntracks from homeassistant.components.owntracks
2020-09-16 10:38:07 INFO (SyncWorker_50) [homeassistant.loader] Loaded konnected from homeassistant.components.konnected
2020-09-16 10:38:07 INFO (SyncWorker_19) [homeassistant.loader] Loaded islamic_prayer_times from homeassistant.components.islamic_prayer_times
2020-09-16 10:38:07 INFO (SyncWorker_14) [homeassistant.loader] Loaded sense from homeassistant.components.sense
2020-09-16 10:38:07 INFO (SyncWorker_1) [homeassistant.loader] Loaded accuweather from homeassistant.components.accuweather
2020-09-16 10:38:07 INFO (SyncWorker_60) [homeassistant.loader] Loaded ecobee from homeassistant.components.ecobee
2020-09-16 10:38:07 INFO (SyncWorker_49) [homeassistant.loader] Loaded guardian from homeassistant.components.guardian
2020-09-16 10:38:07 INFO (SyncWorker_53) [homeassistant.loader] Loaded syncthru from homeassistant.components.syncthru
2020-09-16 10:38:07 INFO (SyncWorker_57) [homeassistant.loader] Loaded local_ip from homeassistant.components.local_ip
2020-09-16 10:38:07 INFO (SyncWorker_55) [homeassistant.loader] Loaded gdacs from homeassistant.components.gdacs
2020-09-16 10:38:07 INFO (SyncWorker_25) [homeassistant.loader] Loaded opentherm_gw from homeassistant.components.opentherm_gw
2020-09-16 10:38:07 INFO (SyncWorker_38) [homeassistant.loader] Loaded upnp from homeassistant.components.upnp
2020-09-16 10:38:07 INFO (SyncWorker_61) [homeassistant.loader] Loaded airvisual from homeassistant.components.airvisual
2020-09-16 10:38:07 INFO (SyncWorker_39) [homeassistant.loader] Loaded smartthings from homeassistant.components.smartthings
2020-09-16 10:38:07 INFO (SyncWorker_23) [homeassistant.loader] Loaded airly from homeassistant.components.airly
2020-09-16 10:38:07 INFO (SyncWorker_2) [homeassistant.loader] Loaded solaredge from homeassistant.components.solaredge
2020-09-16 10:38:07 INFO (SyncWorker_46) [homeassistant.loader] Loaded ring from homeassistant.components.ring
2020-09-16 10:38:07 INFO (SyncWorker_34) [homeassistant.loader] Loaded volumio from homeassistant.components.volumio
2020-09-16 10:38:07 INFO (SyncWorker_40) [homeassistant.loader] Loaded tesla from homeassistant.components.tesla
2020-09-16 10:38:07 INFO (SyncWorker_22) [homeassistant.loader] Loaded monoprice from homeassistant.components.monoprice
2020-09-16 10:38:07 INFO (SyncWorker_0) [homeassistant.loader] Loaded forked_daapd from homeassistant.components.forked_daapd
2020-09-16 10:38:07 INFO (SyncWorker_42) [homeassistant.loader] Loaded nws from homeassistant.components.nws
2020-09-16 10:38:07 INFO (SyncWorker_58) [homeassistant.loader] Loaded agent_dvr from homeassistant.components.agent_dvr
2020-09-16 10:38:07 INFO (SyncWorker_27) [homeassistant.loader] Loaded brother from homeassistant.components.brother
2020-09-16 10:38:07 INFO (SyncWorker_8) [homeassistant.loader] Loaded sms from homeassistant.components.sms
2020-09-16 10:38:07 INFO (SyncWorker_63) [homeassistant.loader] Loaded ambiclimate from homeassistant.components.ambiclimate
2020-09-16 10:38:07 INFO (SyncWorker_41) [homeassistant.loader] Loaded spotify from homeassistant.components.spotify
2020-09-16 10:38:07 INFO (SyncWorker_43) [homeassistant.loader] Loaded openuv from homeassistant.components.openuv
2020-09-16 10:38:07 INFO (SyncWorker_9) [homeassistant.loader] Loaded met from homeassistant.components.met
2020-09-16 10:39:12 WARNING (MainThread) [homeassistant.bootstrap] Setup timed out for stage 2 - moving forward
```
#### Other logs
End of `/var/log/synopkg.log`_
```
2020/09/16 10:31:23	stop homeassistant 0.114.2-7 Begin start-stop-status stop
2020/09/16 10:31:24	stop homeassistant 0.114.2-7 End start-stop-status stop ret=[0]
2020/09/16 10:31:24	stop homeassistant 0.114.2-7 Begin unload apparmor
2020/09/16 10:31:24	stop homeassistant 0.114.2-7 End unload apparmor ret=[0]
2020/09/16 10:31:24	stop homeassistant: stop version 0.114.2-7 successfully, result 0
2020/09/16 10:32:55	start homeassistant: begin to start version 0.114.2-7
2020/09/16 10:32:55	start homeassistant 0.114.2-7 Begin pre-load apparmor
2020/09/16 10:32:55	start homeassistant 0.114.2-7 End pre-load apparmor ret=[0]
2020/09/16 10:32:55	start homeassistant 0.114.2-7 Begin start-stop-status start
2020/09/16 10:33:02	start homeassistant 0.114.2-7 End start-stop-status start ret=[0]
2020/09/16 10:33:02	start homeassistant: start version 0.114.2-7 successfully, result 0
ale@nas:~$ 

```

**Comment by Mr-Groch** [comment]
Default config will not start currently - there are mising wheels for cloud and mobile_app integrations, so you must disable  in your configuration.yaml `default_config:` and manually add all except cloud and mobile_app:



```
#default_config:
# all integrations from default_config:
#cloud:
config:
frontend:
history:
input_boolean:
input_datetime:
input_number:
input_select:
input_text:
logbook:
map:
#mobile_app:
person:
ssdp:
sun:
system_health:
updater:
zeroconf:
zone:
```

**Comment by avol-io** [comment]
works thanks

**Comment by Mr-Groch** [comment]
IMO this should be left opened - issue should  be fixed in package. My answer was just a workaround.

**Comment by avol-io** [comment]
you are right

**Comment by ymartin59** [comment]
May you please detail from where does this "default config" comes from?

**Comment by Mr-Groch** [comment]
After fresh install and initial run - if there will be no manually created configuration.yml, then it will be created automatically with this `default_config:` line (and also with some more defaults, like automations or scripts files)

**Comment by ymartin59** [comment]
OK thanks... As I have understand two different DSM install, I missed that point for initial setup.

**Comment by marcofranssen** [comment]
I don't really get how to resolve the issue on my side. I have freshly installed and am also stuck in same issue atm.

My config currently looks like this.

```yaml
# Configure a default setup of Home Assistant (frontend, api, etc)
default_config:

# Text to speech
tts:
  - platform: google_translate

group: !include groups.yaml
automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml
```

In the logs I found following:

```
Setup failed for cloud: Requirements for cloud not found: ['hass-nabucasa==0.35.0'].
Setup timed out for stage 2 - moving forward
```

What do I need to do specifically to make it work?

**Comment by Mr-Groch** [comment]
You need to change `default_config:` to:

https://github.com/SynoCommunity/spksrc/issues/4182#issuecomment-693302307

**Comment by marcofranssen** [comment]
Unfortunately that doesn't resolve the issue. Config looks like this.

```yaml
# Configure a default setup of Home Assistant (frontend, api, etc)
#default_config:
# all integrations from default_config:
#cloud:
config:
frontend:
history:
input_boolean:
input_datetime:
input_number:
input_select:
input_text:
logbook:
map:
#mobile_app:
person:
ssdp:
sun:
system_health:
updater:
zeroconf:
zone:

# Text to speech
tts:
  - platform: google_translate

group: !include groups.yaml
automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml
```

Is the indentation correct in above comment?

**Comment by Mr-Groch** [comment]
Config is OK now - what is now showing in log?

**Comment by marcofranssen** [comment]
tried couple of restarts, but no new logs coming in when tailing the log.

**Comment by Mr-Groch** [comment]
If nothing is logging, then something else is happening, try looking in syslogs

**Comment by marcofranssen** [comment]
Before I was restarting via the web interface of Home Assistant. Now I stopped via the synology package and started it again I got some logs, all INFO logs in this case. 

It seems all is working now.

So the restart functionality in admin panel of Home Assistant doesn't seem to work on synology package. Is that something we could resolve? I could file a new issue for that.

**Comment by Mr-Groch** [comment]
restrat functionality from frontennd is working - but first full initiall run must be done. Just wait now

**Comment by alistairwitt** [comment]
Could I make a friendly enquiry about any plans to include the missing wheels for cloud and mobile_app integrations? I have a DS218 and would love to use the mobile_app integration in particular. I suspect that the mobile_app integration would be a key one for many people. Would the required wheels be included in an update of the SynoCommunity Python3 package, or the HomeAssistant package?

**Comment by marcofranssen** [comment]
@alistairwitt would love to have the mobile_app integration working on my DS413 as well.

**Comment by hgy59** [comment]
This issue seems to be Home Assistant related as discussed [here](https://community.home-assistant.io/t/home-assistant-is-starting-not-everything-will-be-available-until-it-is-finished/209217)

**Comment by hgy59** [comment]
fixed with Home Assistant Core 0.114.2-9
